src/test/scala/net/ripe/rpki/publicationserver/repository/DataFlusherTest.scala

```diff
@@ -623,86 +584,7 @@ class DataFlusherTest extends PublicationServerBaseTest with Hashing {
     verifyRrpdFiles(sessionId1, serial1)
   }
 
-
-  test("Should update RRDP only when set to do write-rrdp") {
```

Comment:
Why this one is removed?

Reply:
It duplicates other tests in this class, only ensuring that no rsync data is written. Since we never write rsync anymore this test is superfluous.